WTWSMS/common/religions/00_religions.txt

```diff
@@ -1187,6 +1189,46 @@ christian = {
 		religious_clothing_priest = 5
 	}
 	
+	belinist = {
+		graphical_culture = westerngfx
+		
+		color = { 0.23 1.0 0.23 }
+		
+		crusade_name = GREAT_HOLY_WAR
+		scripture_name = THE_BALLADS
+		priest_title = PRIEST
+		
+		high_god_name = GOD_BELI
+		god_names = {
+			GOD_BELI GOD_ESUS GOD_TARANIS
+		}
+		
+		evil_god_names = {
+			SATAN LUCIFER THE_DEVIL
+		}
+		
+		can_excommunicate = no
+		can_grant_divorce = yes
+		can_grant_invasion_cb = invasion
+		can_grant_claim = yes
+		can_call_crusade = no
+		priests_can_inherit = no
+		max_wives = 1
+		
+		can_retire_to_monastery = yes
+		parent = insular
+		bs_marriage = yes
+		pc_marriage = yes
+		religious_clothing_head = 1
+		religious_clothing_priest = 1
+
```

Comment:
Because they're still Christians. Incest was relatively taboo amongst the insulars but not to people of looser faiths like paganism or heresies (De Excidio by Gildas)